README.md

```diff
@@ -1903,7 +1903,7 @@ export default function Component() {
 	// more code here
 	return (
 		// some JSX
-		<HoneypotProvider>
+		<HoneypotProvider {...honeypotInputProps}>
```

Comment:
Took me way too long to realize I was missing this and crucially the honeypot works halfway without it! It'd be wise to help the developer be completely explicit about the context here.